test/Microsoft.AspNetCore.Razor.Evolution.Test/CodeGeneration/RedirectedBasicWriterTest.cs

```diff
@@ -52,7 +52,7 @@ public void WriteCSharpExpression_DesignTime_DoesNormalWrite()
         public void WriteCSharpExpression_Runtime_SkipsLinePragma_WithoutSource()
         {
             // Arrange
-            var writer = new RedirectedBasicWriter(new DesignTimeBasicWriter(), "test_writer")
```

Comment:
These shouldn't really matter here. But I changed them anyway to keep it consistent with what the test is doing.